services/mac_manager/src/main/resources/application.properties

```diff
@@ -1,5 +1,7 @@
+server.port=9005
+#server.port=9000
 #Machine configuration
-ignite.host=localhost
+ignite.host=192.168.137.1
```

Comment:
Shall we use localhost or 127.0.0.1?

Reply:
Yeah I will change it to localhost, it is for docker to test here